typesocket/client.go

```diff
@@ -65,6 +77,33 @@ func (c *ClientConn) Close() error {
 	return c.ws.Close()
 }
 
+func (c *ClientConn) Latency() time.Duration {
+	c.mu.Lock()
+	defer c.mu.Unlock()
+	return c.latency
+}
+
+func (c *ClientConn) StartPings(period time.Duration) {
```

Comment:
Is there a reason to expose this? I'm thinking it could be a private method that gets called in Dial. Perhaps rename to `sendPingsLoop`.

Reply:
Updated